src/codeenginestudio/huong/Exercise4.java

```diff
@@ -0,0 +1,75 @@
+package codeenginestudio.huong;
+
+import java.util.ArrayList;
+
+public class Exercise4 {
+    public static void main(String[] args) {
+        Exercise4 exer4 = new Exercise4();
+        ArrayList<String> ListA = new ArrayList<String>();
```

Comment:
Please use `List` instead of `ArrayList`, ex `List<String> ListA = new ArrayList<String>();`

Reply:
The name of variable should begin by a lower case